packages/plugin-legacy/README.md

```diff
@@ -27,6 +27,12 @@ export default {
 }
 ```
 
+terser must be installed because plugin-legacy uses terser for minification.
```

Comment:
When minification is disabled, `terser` does not need to be installed.
So it should be a optional peer dependency of it?

Reply:
Yeah, technically…
But I can't think of a use case where the user needs to support older browsers but doesn't want their bundles to be as small as possible (except for debugging purposes).

So making it a peer dependency would be more user-friendly, in my opinion.
NPM 7+ would automatically install the peer dependencies. Therefore we would break fewer people's projects.